src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(param) {
```

Comment:
What does it mean `param` ? But better create meaningful variables, `number` for example.

Reply:
done